.changeset/ninety-squids-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/astro": patch
+---
+
+Fix middleware return type.
```

Comment:
Allow the handler of `clerkMiddleware` to return undefined. When undefined is returned `clerkMiddleware` implicitly calls `await next()`

Reply:
thanks Pantelis